jdk/test/jdk/jfr/event/sampling/TestNative.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -ex
+
+OS=`uname -s`
+if [  "${OS}" != "Linux" -a "${OS}" != "Darwin" ]; then
+    echo "This is a Linux/MacOSX only test"
+    exit 0;
+fi
+
+if [ "x$TESTGCC" = "x" ]; then
+  TESTGCC=$(readlink -f $(which gcc))
+fi
+
+if [ "x$TESTGCC" = "x" ]; then
+  echo "WARNING: gcc not found. Cannot execute test." 2>&1
+  exit 1;
+fi
+
+if [ "x$TESTROOT" = "x" ]; then
+  echo "TESTROOT pointintg to top level sources is not set. that is fatal"
+  exit 2;
+fi
+
+if [ "x$TESTJAVA" = "x" ]; then
+  TESTJAVA=$(dirname $(dirname $(readlink -f $(which java))))
+fi
+
+JDK_TOPDIR="${TESTROOT}/.."
+JAVAC="${TESTJAVA}/bin/javac"
+JAVA="${TESTJAVA}/bin/java"
+TEST_ENV_SH="${JDK_TOPDIR}/../hotspot/test/test_env.sh"
+
+ls -l "${TEST_ENV_SH}"
```

Comment:
This looks like a debug aid. Please remove before pushing.

Reply:
Nope. It is here to fail the test if the file is missing.
See below, the +e/ source -e - the test_env.sh is not designed to run with -e. 

Ok to go  with this in?

Reply:
Fair enough. Feel free to proceed.